src/adapters/github.less

```diff
@@ -124,13 +124,13 @@
         .octotree-header-repo,
         .octotree-header-branch {
           width: 100%;
-          overflow: hidden;
```

Comment:
Why delete this? Thought this is needed in case branch name is long.

Reply:
Oh, my fault.

I think we should change `top: 0px` in `.octotree-icon-branch`(or `padding-bottom: 1px;` `.octotree-header-branch`)